packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -661,7 +661,7 @@ const SelectButton = React.forwardRef((props, forwardedRef) => {
       data-iui-variant={styleType !== 'default' ? styleType : undefined}
       {...rest}
       ref={forwardedRef}
-      className={cx('iui-select-button', props.className)}
+      className={cx('iui-select-button iui-field', props.className)}
```

Comment:
This can be written as a list for clarity. Similar to .

Same comment for the other places too.

Reply:
Done.  I caught 1 in toast that looks like it came from someone else's PR, ignored that 1.